<launch>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->

  <node pkg="joy_controller"
    type="teleop_joyphid" name="teleop"/>
  <param name="velocity_ab" type="double" value="1.0" />
  <param name="velocity_xy" type="double" value="2.0" />
  <param name="velocity_axis" type="double" value="1.0" />
  <param name="axis_treshold" type="double" value="0.8" />


  <node pkg="joy_controller"
    type="joyphid_float64" name="joy_float64" />


    <node name="dynamixel_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
        <rosparam>
            namespace: dxl_manager
            serial_ports:
                pan_tilt_port:
                    port_name: "/dev/ttyUSB0"
                    baud_rate: 57600
                    min_motor_id: 1
                    max_motor_id: 25
                    update_rate: 20
        </rosparam>
    </node>
    <!-- Start pan joint controller -->
    <rosparam file="$(find my_dynamixel_tutorial)config/pan.yaml" command="load"/>
    <node name="pan_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port pan_tilt_port
                pan_controller"
          output="screen"/>

</launch>
